code/game/machinery/Sleeper.dm

```diff
@@ -272,10 +272,7 @@
 				if(beaker.reagents.total_volume < beaker.reagents.maximum_volume)
 					var/pumped = 0

```

Comment:
Variable defined but not used.

Reply:
Used a few lines up

Reply:
The var a few lines up is a different one with the same name, defined at L261